addons/web_tour/static/src/tour_service/tour_compilers.js

```diff
@@ -642,7 +642,7 @@ export function compileStepAuto(stepIndex, step, options) {
                 browser.clearTimeout(tourTimeout);
                 tourState.set(tour.name, "currentIndex", stepIndex + 1);
 
-                if (showPointerDuration > 0) {
+                if (showPointerDuration > 0 && stepEl instanceof Node) {
```

Comment:
if stepEl is coming from an Iframe, this will be false.
for testing if an object responds to the node's API, i'd suggest using nodeType https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType

Reply:
Error in my PR ... the solution was not this one ... sorry